src/styles.css

```diff
@@ -60,15 +60,35 @@ body {
   color: #91caff;
 }
 
+
+/* BEGIN about HTML styling */
+
+/*
+Header elements which are the first child of the about content container shouldn't have the default Ant Design top
```

Comment:
The about html does not inherit antd styles for what I know so this shouldnt be an issue.

Reply:
it does seem to on my end and is visible for example with RENATA: https://renata.bento.sd4h.ca/en/